### DISCUSSION TITLE
Add a btcpayserver payment provider

### DISCUSSION THREAD

**Discussion by Zoz92** [pretix/pretix#3225] [Ideas] [answered]
Hello, i try to use some crypto-currency for paid tickets, i wonder if you could implement [btcpayserver](https://btcpayserver.org/) who is open source project as a payment provider ?

**Comment by pc-coholic** [comment, marked as answer]
Please use the [pretix-bitpay](https://github.com/pretix/pretix-bitpay) plugin.

BTCPayServer uses the same protocol, so it can be used with it

**Comment by Zoz92** [comment]
Oh thanks i will give a try on my instance.

**Comment by Zoz92** [comment]
Any advice to do it with a docker environnement ?

**Reply by pc-coholic** [reply]
Please check out the documentation for that: https://docs.pretix.eu/en/latest/admin/installation/docker_smallscale.html#install-a-plugin

**Reply by Zoz92** [reply]
Nice , it is working perfectly thanks @pc-coholic
